pkg/core/resources/apis/mesh/external_service_helpers_test.go

```diff
@@ -0,0 +1,41 @@
+package mesh
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	mesh_proto "github.com/kumahq/kuma/api/mesh/v1alpha1"
+	system_proto "github.com/kumahq/kuma/api/system/v1alpha1"
+	util_proto "github.com/kumahq/kuma/pkg/util/proto"
+)
+
+var _ = Describe("ExternalServiceResource", func() {
+	Describe("MarshalLog", func() {
+		It("should marshal log without panicking", func() {
```

Comment:
Please change the name to something like `It("should mask the sensitive information when marshaling")`